### PR TITLE
Create a hydra-js package for volto-hydra

### PR DESCRIPTION
**Pull request by MAX-786**

### CODE REVIEW

File: packages/hydra-js/package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "name": "@volto-hydra/hydra-js",
+  "version": "1.0.0",
+  "description": "Provides a method to initialize a bridge which enables communication between Volt Hydra and your frontend",
+  "private": true,
+  "exports": {
+  ".": {
+    "import": "./hydra.js",
+    "require": "./hydra.js"
+  }
+},
```
Comment (JeffersonBledsoe) [inline packages/hydra-js/package.json:11]: @MAX-786 Formatting is off. Also, you only need separate `import` and `require` if you have a separate `.cjs` or `.mjs` file. If its the same file you, can just use a string
```suggestion
  ".": "./hydra.js"
},
```

**Review by JeffersonBledsoe — COMMENTED** [review]
@MAX-786 Just the one minor thing, thanks for doing this :)

**Comment by JeffersonBledsoe** [comment]
@MAX-786 Any imports will need to use the package from now on. Nothing in the `volto-hydra` addon use this file right? It's all to be used on the integration side?

**Comment by MAX-786** [comment]
I am ready to create a draft for block's border which will be changing hydra.js, so can this be merged now

**Comment by MAX-786** [comment]
> Nothing in the volto-hydra addon use this file right? It's all to be used on the integration side?

Yepp, its needed on frontend only

**Pull request merged**